src/components/DevModeLogin.js

```diff
@@ -28,12 +29,17 @@ class DevModeLogin extends React.Component {
         '/login/dev',
         {
           netid: this.state.netid,
+          redirect: this.props.redirect,
         },
         {
           withCredentials: true,
         }
       )
-      .then(() => Router.pushRoute('index'))
+      .then(res => {
```

Comment:
I'm not sure if doing this dance between the server and client is really necessary - it should just be safe to do `window.location = this.props.redirect` after the login request completes. Is there a particular reason you chose this approach?